content/en/docs/reference/instrumentation/slis.md

```diff
@@ -0,0 +1,91 @@
+---
+reviewers:
+- logicalhan
+title: Kubernetes Component SLIs
+content_type: reference
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+The Component SLIs feature allows enabling of a new metrics endpoint for each Kubernetes
+binary. This metric endpoint is exposed on the standard port of each binary at the endpoint
+`metrics/slis`. 
+
+<!-- body -->
+
+## SLI Metrics
+
+We expose two metrics; a gauge (which represents the current state of the healthcheck) and a
+counter (which records the cumulative counts that we've observed for each healthcheck state).
+
+```golang
+var (
+   // healthcheck is a Prometheus Gauge metrics used for recording the results of a k8s healthcheck.
+   healthcheck = k8smetrics.NewGaugeVec(
+      &k8smetrics.GaugeOpts{
+         Namespace:      "kubernetes",
+         Name:           "healthcheck",
+         Help:           "This metric records the result of a single healthcheck.",
+         StabilityLevel: k8smetrics.ALPHA,
+      },
+      []string{"name", "type"},
+   )
+
+   // healthchecksTotal is a Prometheus Counter metrics used for counting the results of a k8s healthcheck.
+   healthchecksTotal = k8smetrics.NewCounterVec(
+      &k8smetrics.CounterOpts{
+         Namespace:      "kubernetes",
+         Name:           "healthchecks_total",
+         Help:           "This metric records the results of all healthcheck.",
+         StabilityLevel: k8smetrics.ALPHA,
+      },
+      []string{"name", "type", "status"},
+   )
+)
+```
+
+The gauge data looks like this:
+
+```shell
+# HELP kubernetes_healthcheck [ALPHA] This metric records the result of a single healthcheck.
+# TYPE kubernetes_healthcheck gauge
+kubernetes_healthcheck{name="autoregister-completion",type="healthz"} 1
+kubernetes_healthcheck{name="autoregister-completion",type="readyz"} 1
+kubernetes_healthcheck{name="etcd",type="healthz"} 1
+kubernetes_healthcheck{name="etcd",type="readyz"} 1
+kubernetes_healthcheck{name="etcd-readiness",type="readyz"} 1
+kubernetes_healthcheck{name="informer-sync",type="readyz"} 1
+kubernetes_healthcheck{name="log",type="healthz"} 1
+kubernetes_healthcheck{name="log",type="readyz"} 1
+kubernetes_healthcheck{name="ping",type="healthz"} 1
+kubernetes_healthcheck{name="ping",type="readyz"} 1
+```
+
+While the counter data looks like this:
+
+```shell
```

Comment:
(nit)
This is actually OpenMetrics (or something close), not shell. It's OK to omit a highlighting hint. Please don't set one that's wrong though.

Reply:
It colorizes pretty correctly under this syntax though. It looks nicer than if I omit here.

Reply:
removed